uk.co.compucorp.civicrm.tasksassignments/js/test/services/assignment_test.js

```diff
@@ -36,5 +39,29 @@ define([
         }));
       });
     });
+
+    describe('get()', function () {
+      beforeEach(function () {
+        param = {
+          json: {
+            options: {
+              limit: 0
+            },
+            id: {
+              IN: [ 1, 2 ]
+            },
+            return: [ 'case_type_id', 'contacts', 'client_id', 'contact_id', 'id', 'is_deleted', 'start_date', 'status_id', 'subject' ],
+            debug: true
+          }
+        };
+
+        spyOn(Assignment, 'get').and.callThrough();
+        AssignmentService.get({ 'IN': [1, 2] });
+      });
+
+      it('calls Assignment.get with correct list of fields to be returned', function () {
```

Comment:
you are just testing if the `return` field has a certain value, why do you need the `param` at all? You are just adding a log of noise that takes focus on what you are actually testing

```js
describe('get()', function () {
  beforeEach(function () {
    spyOn(Assignment, 'get').and.callThrough();
    AssignmentService.get({ 'IN': [1, 2] });
  });

  it('<title>', function () {
    expect(Assignment.calls.mostRecent().args[0]).toEqual(<array of fields>);
  });
});
```